Source/Rows/Common/FieldRow.swift

```diff
@@ -381,6 +391,30 @@ open class _FieldCell<T> : Cell<T>, UITextFieldDelegate, TextFieldCell where T:
 
     open func textFieldShouldEndEditing(_ textField: UITextField) -> Bool {
         return formViewController()?.textInputShouldEndEditing(textField, cell: self) ?? true
-    }
-
+	}
+	
+	open override func layoutSubviews() {
+		super.layoutSubviews()
+		guard let row = (row as? FieldRowConformance) else { return }
+		guard let titlePercentage = row.titlePercentage else {
+			calculatedTitlePercentage = nil
+			return
+		}
+		var targetTitleWidth = bounds.size.width * titlePercentage
+		if let imageView = imageView, let _ = imageView.image, let titleLabel = titleLabel {
+			var extraWidthToSubtract = titleLabel.frame.minX - imageView.frame.minX // Left-to-right interface layout
+			if #available(iOS 9.0, *) {
```

Comment:
Why don't use `UIApplication.shared.userInterfaceLayoutDirection == .rightToLeft` to determine layout direction in iOS 8 ?